Buildings/Templates/Components/Validation/Coils.mo

```diff
@@ -122,17 +122,18 @@ model Coils "Validation model for coil components"
       iconVisible=false,
       transformation(extent={{-20,-100},{20,-60}}),
       iconTransformation(extent={{-250,-32},{-210,8}})));
-  Controls.OBC.CDL.Reals.Sources.Constant TOut(k=coiEva.dat.datCoi.sta[1].nomVal.TConIn_nominal)
+  Buildings.Controls.OBC.CDL.Reals.Sources.Constant TOut(
+    k=coiEva.dat.datCoi.sta[1].nomVal.TConIn_nominal)
     "Outdoor temperature"
     annotation (Placement(transformation(extent={{-130,-110},{-110,-90}})));
-  Controls.OBC.CDL.Reals.Sources.Constant XOut(k=0.015)
+  .Buildings.Controls.OBC.CDL.Reals.Sources.Constant XOut(k=0.015)
```

Comment:
Is there any reason to add the dot in `.Buildings`, here and on the lines below.

Reply:
No reason: it seems that Dymola uses "globally scoped" class names like this when it automatically updates class names to avoid conflicts resulting from a newly created package that shadows an existing package (here `Buildings.Templates.Components.Controls` collides with `Buildings.Controls`).
I have changed these class names without the starting dot.